EIPS/eip-5757.md

```diff
@@ -0,0 +1,94 @@
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Definitions
+
+ - **Link**: Any method of referring to a resource, including: markdown links, anchor tags (`<a>`), images, citations of books/journals, and any other method of referencing content not in the current resource.
+ - **Resource**: A web page, document, file, or other media that contains content.
+ - **Source**: A publisher/chronicler of resources, like a standards body (eg. w3c) or a system of referring to documents (eg. Digital Object Identifier System).
+
+### Requirements for Sources
+
+Permissible sources **SHOULD NOT** obviously allow redistributing their resources as part of the EIP repository. If a source does obviously allow redistribution of resources, include those resources in an appropriate directory within this repository and link to those copies instead. The common exception to this rule are git repositories under the Ethereum organization.
+
+Permissible sources **MUST** provide a method of uniquely identifying a particular revision of a resource. Examples of such methods may include git commit hashes, version numbers, or publication dates.
+
+Permissible sources **MUST** have a proven history of availability. A source existing for at least ten years and reliably serving resources would be sufficient—but not necessary—to satisfy this requirement.
+
+Permissible sources **MUST NOT** charge a fee for accessing resources.
+
+### Source Removal
+
+Any approved source that ceases to satisfy the above requirements **MUST** be removed from [EIP-1](./eip-1.md), and its proposal (as described below) **MUST** be moved to a `status` of `Draft` or `Withdrawn`.
+
+Finalized EIPs (eg. those in the `Final` or `Withdrawn` statuses) **SHOULD NOT** be updated to remove links to these sources.
+
+Non-Finalized EIPs **MUST** remove links to these sources before changing statuses.
+
+### Source Approval
+
+#### Proposal
+
+Each potential permitted source **SHALL** be described in its own EIP.
+
+These EIPs **MUST** describe how the source meets the requirements described above.
+
+These EIPs **MUST** include the markup for creating a link that satisfies all the above requirements and an example of such a link.
+
+When opened, these EIPs:
+
+ - **MUST** have a `status` of `Draft`.
+
+Throughout their lifetime, these EIPs:
+
+ - **MUST** have a `type` of `Meta`;
+ - **MUST** have a `title` beginning with `External Source - ` followed by a unique name describing the source; and
+ - **MAY** include links to any resources relevant to the approval process (regardless of source).
+
+[A template for these proposals is available](../assets/eip-5757/template.md).
```

Comment:
I think in order to make this work we have to implement #4335 first.

Reply:
> That list will be extremely long.

I sincerely hope not :rofl:

Reply:
At the current rate of things, it'll probably be easier to do taxes than figure out if an external link is allowed.